clients/src/test/java/org/apache/kafka/clients/consumer/StickyAssignorTest.java

```diff
@@ -606,6 +606,22 @@ public void testStickiness() {
         }
     }
 
+    @Test
+    public void testAssignmentUpdatedForDeletedTopic() {
+        String consumerId = "consumer";
+
+        Map<String, Integer> partitionsPerTopic = new HashMap<>();
+        partitionsPerTopic.put("topic01", 1);
+        partitionsPerTopic.put("topic03", 100);
+        Map<String, Subscription> subscriptions =
+                Collections.singletonMap(consumerId, new Subscription(Arrays.asList("topic01", "topic02", "topic03")));
```

Comment:
nit: You can use the `topic(...)` helper function instead of `Arrays.asList(...)`. See examples in other unit tests.